app/src/main/java/org/connectbot/ConsoleActivity.java

```diff
@@ -87,6 +74,20 @@
 import android.widget.RelativeLayout;
 import android.widget.TextView;
 import android.widget.Toast;
+
+import org.connectbot.bean.HostBean;
+import org.connectbot.service.BridgeDisconnectedListener;
+import org.connectbot.service.PromptHelper;
+import org.connectbot.service.TerminalBridge;
+import org.connectbot.service.TerminalKeyListener;
+import org.connectbot.service.TerminalManager;
+import org.connectbot.util.PreferenceConstants;
+import org.connectbot.util.TerminalViewPager;
+
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
+import java.util.List;
+
```

Comment:
Corrected, sorry about that.
Everytime I touch code and/or xml in ConnectBot I'm struggling with Android Studio and my default settings. Indent with spaces is systematically selected for me in **all** code style configs and imports are automatically rearrange and optimized.